doc/support.rst

```diff
@@ -102,20 +102,19 @@ questions.
 Gitter
 ======
 
-**Note**: The scikit-learn Gitter room is no longer an active community. 
-For live discussions and support, please refer to the other channels 
+**Note**: The scikit-learn Gitter room is no longer an active community.
+For live discussions and support, please refer to the other channels
 mentioned in this document.
 
 .. _documentation_resources:
 
 Documentation Resources
 =======================
 
-This documentation is for |release|. Find documentation for other versions 
+This documentation is for |release|. Find documentation for other versions
 `here <https://scikit-learn.org/dev/versions.html>`__.
 
-Older versions' printable PDF documentation is available `here
-<https://sourceforge.net/projects/scikit-learn/files/documentation/>`_.
-Building the PDF documentation is no longer supported in the website,
-but you can still generate it locally by following the
-:ref:`building documentation instructions <building_documentation>`.
+The printable PDF documentation is available for 0.23.2 (released in August 2020) `here
+<https://scikit-learn.org/0.23/_downloads/scikit-learn-docs.pdf>`__.
```

Comment:
Nitpick:

```suggestion
Documentation in the PDF format is available for 0.23.2 (released in August 2020) `here
<https://scikit-learn.org/0.23/_downloads/scikit-learn-docs.pdf>`__.
```

An edit I'd really like but can't quickly think of is to make it more obvious that "the PDF you will get is really really old. Unless you are an archeologist you probably don't want to look at it."

I'd remove "printable" because you can also print HTML pages, maybe not as one big document but individual pages don't look terrible.

Maybe there is someone who understands CSS well enough to help improve the "printer" stylesheet?? Worth making an issue or too niche? I'd almost think too niche

Reply:
I pushed another commit, which puts the PDF link even later, let me know if this is going in the right direction ...